src/main/java/entities/UploadObject.java

```diff
@@ -0,0 +1,41 @@
+package entities;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Random;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
```

Comment:
Hey Zack, great work here ! It would be helpful if you could share the link to this library and add it to the build.gradle file for compilation purposes.

**UPDATE:** My apologies, I see that you have updated the build.gradle !